_posts/2025-03-25-costowell-nautilus.md

```diff
@@ -8,15 +8,15 @@ categories:
 author: Cole Stowell
 ---
 
-Its really easy to contribute to GNOME! Here's how I did it.
+It's easy to contribute to GNOME! Here's how I did it.
 
 ## The GNOME Community
 
 I picked Nautilus, the GNOME project's file manager, for my contribution.
 I use Nautilus *every single day* and because I love writing C,
 it was the perfect project to start contributing to.
 
-GNOME is a federation type open source community, meaning there is a
+GNOME is a federation-type open-source community, meaning there is a
```

Comment:
open source should not be hyphenated

https://opensource.org/blog/is-open-source-ever-hyphenated

Reply:
Ahhh, I ran it through Grammarly since that's one of the requirements and it changed that...
Should be fixed, thanks!